apps/src/blocklyAddons/cdoWorkspaceSvg.js

```diff
@@ -70,3 +70,22 @@ WorkspaceSvg.prototype.blockSpaceEditor = {
   },
   svgResize: () => {} // TODO
 };
+
+/** Force content to start in top-left corner, not scroll in all directions.
+ * @override
+ */
+WorkspaceSvg.getContentDimensionsBounded_ = function(ws, svgSize) {
+  const content = Blockly.WorkspaceSvg.getContentDimensionsExact_(ws);
+
+  // View height and width are both in pixels, and are the same as the SVG size.
+  const viewWidth = svgSize.width;
+  const viewHeight = svgSize.height;
+
+  const dimensions = {
+    left: 0,
+    top: 0,
+    height: Math.max(content.bottom + viewHeight / 2, viewHeight),
```

Comment:
Out of curiosity, what is this doing?

Reply:
the first term is the length of the blocks in the workspace + a little bit (this is how we make more room as your code gets close to the bottom). The second term is just the height of the visible space. The workspace height should be the max of these two. In other words, the workspace fits the height of its container, unless the code is long, in which case it is the height of the code plus a buffer.

Reply:
Interesting!  Given there is a big of artistic license there, maybe a comment to capture the thinking?